Normal scenarios typicaly deal with 1 or 2 bridge
domains. However opensync networks are a lot more
complex, with multiple single-interface bridges
(as far as eapd/nas logic is concerned), eg.

 wl0 is a sta, so it needs a separate lan_ifname
 wl0.1 is we.backhaul so it needs a separate lan_ifname
 wl0.2 is home bss so it goes to a shared lan_ifname=br-home
 wl0.3 is we.piranha onboard so it needs a separate lan_ifname

This is repeated for each radio. For tri-band
that'd be 12. So it makes sense to preallocate a
bit more for flexibility.
--- a/bcmdrivers/broadcom/net/wl/impl29/main/src/router/shared/wlif_utils.h
+++ b/bcmdrivers/broadcom/net/wl/impl29/main/src/router/shared/wlif_utils.h
@@ -37,10 +37,10 @@
 #endif // endif
 
 #ifdef DSLCPE
-#define WLIFU_MAX_NO_BRIDGE   8
+#define WLIFU_MAX_NO_BRIDGE   32
 #define WLIFU_MAX_NO_WAN      8
 #else
-#define WLIFU_MAX_NO_BRIDGE		2
+#define WLIFU_MAX_NO_BRIDGE		32
 #define WLIFU_MAX_NO_WAN		2
 #endif
 
--- a/bcmdrivers/broadcom/net/wl/impl32/main/src/router/shared/wlif_utils.h
+++ b/bcmdrivers/broadcom/net/wl/impl32/main/src/router/shared/wlif_utils.h
@@ -34,10 +34,10 @@
 #endif // endif
 
 #ifdef DSLCPE
-#define WLIFU_MAX_NO_BRIDGE   8
+#define WLIFU_MAX_NO_BRIDGE   32
 #define WLIFU_MAX_NO_WAN      8
 #else
-#define WLIFU_MAX_NO_BRIDGE	2
+#define WLIFU_MAX_NO_BRIDGE	32
 #define WLIFU_MAX_NO_WAN		2
 #endif
 
--- a/bcmdrivers/broadcom/net/wl/impl61/main/components/router/shared/wlif_utils.h
+++ b/bcmdrivers/broadcom/net/wl/impl61/main/components/router/shared/wlif_utils.h
@@ -36,7 +36,7 @@
 #define ETHER_ADDR_LEN 6
 #endif // endif
 
-#define WLIFU_MAX_NO_BRIDGE		2
+#define WLIFU_MAX_NO_BRIDGE		32
 
 #define WLIFU_MAX_NO_WAN		2
 
--- a/bcmdrivers/broadcom/net/wl/impl63/main/components/router/shared/wlif_utils.h
+++ b/bcmdrivers/broadcom/net/wl/impl63/main/components/router/shared/wlif_utils.h
@@ -36,7 +36,7 @@
 #define ETHER_ADDR_LEN 6
 #endif // endif
 
-#define WLIFU_MAX_NO_BRIDGE		2
+#define WLIFU_MAX_NO_BRIDGE		32
 
 #define WLIFU_MAX_NO_WAN		2
 
